data/sles4sap/qe_sap_deployment/sles4sap_azure_generic.yaml

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Generic yaml template for use with qe-sap-deployment project.
+#   Settings are meant to be controlled via OpenQA variables and managed by test:
+#   tests/sles4sap/publiccloud/qesap_ansible.pm
+# https://github.com/SUSE/qe-sap-deployment
+
+provider: 'azure'
+apiver: 2
+terraform:
+  provider: 'azure'
+  variables:
+    # GENERAL VARIABLES #
+    az_region: "%PUBLIC_CLOUD_REGION%"
+    admin_user: "cloudadmin"
+    public_key : "~/.ssh/id_rsa.pub"
+    private_key: "~/.ssh/id_rsa"
+    sles4sap_uri: "https://%STORAGE_ACCOUNT_NAME%.blob.core.windows.net/sle-images/%SLE_IMAGE%"
+    hana_os_major_version: "%HANA_OS_MAJOR_VERSION%"
+    storage_account_name: "%STORAGE_ACCOUNT_NAME%"
+    storage_account_key: "%STORAGE_ACCOUNT_KEY%"
+
+    # HANA
+    hana_count: "%NODE_COUNT%"
+    hana_ha_enabled: "%QESAP_HANA_HA_ENABLED%"
+    vm_size: "%QESAP_VM_SIZE%"
+    hana_cluster_fencing_mechanism: "%FENCING_MECHANISM%"
+
+ansible:
+  hana_urls:
+    - "%HANA_SAR%"
+    - "%HANA_CLIENT_SAR%"
+    - "%HANA_SAPCAR%"
+  destroy:
+    - deregister.yaml
```

Comment:
So this YAML is special as the _configure:_ playbooks list is missing. Right?

Reply:
Yep. It's a bit quirky to add playbook (or any other) list into config using the %SOMETHING% method.
So I am doing it directly in YAML. Ideally it would be to do the whole thing like that and there will be no need for yaml template. But having deployment is priority so we can think about it in the future.